src/views/pages/student.jsx

```diff
@@ -32,11 +33,15 @@ export default class Student extends ListenerComponent {
         checkOutItems(this.state.student.id, this.state.itemAddresses);
     }
 
+    cancel() {
+        clearCart();
```

Comment:
Is there a benefit to moving `clearCart` into the API client? It seems like the API client would only be concerned with making API calls. We could just import the Dispatcher into here and call `Dispatcher.handleAction('CLEAR_ITEMS');` directly.